js/src/data/test/reducer.test.js

```diff
@@ -418,87 +413,50 @@ describe( 'reducer', () => {
 	} );
 
 	describe( 'Policy Check', () => {
-		it( 'should return with allowed countries', () => {
+		it( 'should return with policy check info', () => {
```

Comment:
Since the reducer only takes care of placing the received data of the `TYPES.POLICY_CHECK` action to a specific position in the data store tree, this whole test case could be simplified by adding a new tuple to `[ TYPES.POLICY_CHECK, 'data', 'mc.policy_check' ],` to the below test case:

https://github.com/woocommerce/google-listings-and-ads/blob/0197013965630d1563fb65119f93f9227b00c262/js/src/data/test/reducer.test.js#L775-L793

Reply:
Thanks for your suggestions! Updated.